app/tools/variable_evaluator.rb

```diff
@@ -63,8 +63,8 @@ def substitutions(subs_hash)
         'je_begeleidingsinitiatief' => subs_hash[:organization],
         'deze_student' => subs_hash[:student_name],
         'achternaam_student' => subs_hash[:student_last_name],
-        'datum_lang' => I18n.l(Date.today, format: :long),
-        'datum' => I18n.l(Date.today),
+        'datum_lang' => I18n.l(Time.zone.today, format: :long),
```

Comment:
same here

Reply:
Ja dit is hetzelfde, maar dan gelocalized met timezone. Dat moet nu van rubocop:

![vsv — be rails c — be — ruby ◂ be rails c — 111×61 2019-06-11 13-49-15](https://user-images.githubusercontent.com/607965/59269601-bba57a80-8c4f-11e9-8299-e3b9b87b8f2f.png)